src/services/CalculatorProcess.js

```diff
@@ -1,51 +1,52 @@
-let memory = [];
-let result = "";
-function calculatorProcess(content,visor) {
-	
-	const numbersOperators = () => String(eval((result)));
-	
-	memory += content;
+let memory = "";
+let sLength;
 
-	if (memory.includes("=")) {
+function CalculatorProcess(content, visor) {
 
-		result = memory.slice(0,-1);
+	const firstInteraction = ["%", "/", "*", "+", "-", "S", "C", "="];
 
-		if (memory.includes("+")) {
-			memory = numbersOperators();
-		}
+	const conversion = (string) => String(eval(string));
 
-		if (memory.includes("-")) {
-			memory = numbersOperators();
-		}
+	memory += content;
```

Comment:
there is an issue in your logical 👀 

I guess you can just catch up this, if you right your logical line by line (I don't mean copy and past the same logical, I mean you forgot about this logical and write again just what is necessary for your tests pass)

Reply:
ok think i got, you mean i write all over again just with what it's needed.
then create new tests for the other new scenarios that could happen ?

Reply:
because i didn't created new tests for the new code lines I've written, right ?

Reply:
i don't mean new tests, i meant new scenarios for the test i already have

Reply:
i written the all over again, but the logical i was thinking don't changed that much.  :/

Reply:
done